bears/general/CasingBear.py

```diff
@@ -0,0 +1,153 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.naming_conventions import (
+    to_camelcase, to_pascalcase, to_snakecase)
```

Comment:
worth checking whether `isort` for PyImportSortBear accepts this convention ? @AbdealiJK may know the answer..